### DISCUSSION TITLE
How to input a {{}} as input parameters for command?

### DISCUSSION THREAD

**Discussion by GaoJianAllen** [go-task/task#1681] [Q&A] [answered]
I have a task contains {{}}:
> gomarkdoc:
>   desc: Generate readme document
>   cmds:
>     - bin/gomarkdoc --output '{{.Dir}}/README.md' ./...

But the command will render to "bin/gomarkdoc --output '/README.md' ./..." , how to ignore the template render?

**Comment by pd93** [comment, marked as answer]
https://stackoverflow.com/questions/17641887/how-do-i-escape-and-delimiters-in-go-templates
